src/components/HomePage.js

```diff
@@ -129,6 +179,9 @@ function HomePage() {
           <div>
             <label>Are you sure, you want to remove the job?</label>
             <label>This action is irreversible!</label>
+            <Button variant="danger" onClick={handleDeleteJob}>
```

Comment:
Here, we don't need to add separate OK button, we can use the common OK and Cancel buttons available in Modal footer.

Reply:
Resolved the Issue

Reply:
<img width="1680" alt="Screenshot 2024-09-10 at 9 02 56 PM" src="https://github.com/user-attachments/assets/1cac49f8-0183-4443-8aac-4298a7182d45">
When a duplicate job is added, the current error message displays as "Error adding job." I believe we can enhance the clarity by updating the message to something more specific, such as "Duplicate job detected."